src/clients/ShieldAssociationClient.js

```diff
@@ -186,7 +186,7 @@ ShieldAssociationClient.prototype.getShieldAssociationsPerUser = function (usern
 
   var parameters = {
     options: {
-      url: this.baseUrl + '/shieldassociation/byuser/' + username,
+      url: this.baseUrl + '/shieldassociation/byUser/' + username,
```

Comment:
@patricklx I think we should change the unit test to match this not the other way around. The API signature is correct "GET /shieldassociation/byuser/{userName}"